### PR TITLE
Added CoinCorner Level 4 Wallet

### PR DESCRIPTION
**Pull request by brollymook**
PNG optimized for web

If I've missed anything off or something isn't up to scratch let me know!

### CODE REVIEW

**Comment by brollymook** [comment]
@harding is there anything more needed on this? can I help out at all?

File: _templates/choose-your-wallet.html [hunk 1]
```diff
@@ -685,6 +685,28 @@
...
           privacynetwork: "checkpassprivacynetworksupporttorproxy"
+          
+- coincorner:
+    title: "CoinCorner"
+    titleshort: "CoinCorner"
+    compat: "web"
+    level: 4
+    platform:
+      web:
+        text: "walletcoincoirner"
```
Comment (schildbach) [inline _templates/choose-your-wallet.html]: typo

**Comment by schildbach** [comment]
Imho the screenshot should not show a device frame.

**Comment by brollymook** [comment]
@schildbach thanks for the feedback... amended raised issues.

**Comment by crwatkins** [comment]
Note that this is currently tagged Help Needed as explained in #1109

**Comment by wbnns** [comment]
@brollymook Hello, if you'd still like to potentially be added to the site, please resubmit a new PR for CoinCorner to be added to the [Bitcoin Exchanges](https://bitcoin.org/en/exchanges) page.

**Pull request closed without merging**